### ISSUE TITLE
Getting all rockets doesn't retrieve wikipedia links

### ISSUE THREAD

**Issue by jesusrp98** [r-spacex/SpaceX-API#121]
Calling a rocket individually does retrieve the rocket's Wikipedia article link, but when you retrieve all rocket at once, they doesn't include the Wikipedia link.

Does include link:
```
https://api.spacexdata.com/v2/rockets/falcon1
```
Doesn't include link:
```
https://api.spacexdata.com/v2/rockets
```

**Comment by jakewmeyer** [comment]
More than likely a cache issue. The rocket info doesn't change often, so I have redis cache it for 24 hours. The cache might have refreshed at different times, causing the issue

**Comment by jesusrp98** [comment]
Yeah! After a couple hours after I post the issue it started working! Thanks :)
